operations/restrictions.md

```diff
@@ -468,7 +468,106 @@ This reduces the overall number of restrictions and improves system performance.
+
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/restrictions/clear`
+
+```javascript
+{  
+   "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+   "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+   "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+   "Data": [  
+      {  
+         "Type": "Start",
+         "StartUtc": "2023-02-15T00:00:00Z",
+         "EndUtc": "2023-02-22T00:00:00Z",
+         "ExactRateId": "7c7e89d6-69c0-4cce-9d42-35443f2193f3",
+         "ResourceCategoryId": "86336EAC-4168-46B1-A544-2A47251BF864",
+         "Days": {
+            "Monday": false,
+            "Tuesday": false,
+            "Wednesday": false,
+            "Thursday": false,
+            "Friday": true,
+            "Saturday": true,
+            "Sunday": true
+         }
+      },
+      {
+         "Type": "Start",
+         "StartUtc": "2023-02-23T00:00:00Z",
+         "EndUtc": "2023-03-03T00:00:00Z",
+         "BaseRateId": "e5b538b1-36e6-43a0-9f5c-103204c7f68e",
+         "Days": {
+            "Monday": false,
+            "Tuesday": false,
+            "Wednesday": false,
+            "Thursday": false,
+            "Friday": true,
+            "Saturday": true,
+            "Sunday": true
+         }
+      }
+   ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) restrictions will be set in. |
+| `Data` | Array of [Restriction clear data](#restriction-clear-data) | required | Clear parameters. |
+
+#### Restriction clear data
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Type` | string | required | [Restriction type](#restriction-type). |
+| `ExactRateId` | string | optional | Unique identifier of the exact [Rate](rates.md#rate) of the restrictions to be spliced. |
+| `BaseRateId` | string | optional | Unique identifier of the base [Rate](rates.md#rate) of the restrictions to be spliced. |
+| `RateGroupId` | string | optional | Unique identifier of the [Rate group](rates.md#rate-group) of the restrictions to be spliced. |
+| `ResourceCategoryId` | string | optional | Unique identifier of the [Resource category](resources.md#resource-category) of the restrictions to be spliced. |
+| `ResourceCategoryType` | string | optional | Name of the [Resource category type](resources.md#resource-category-type) of the restrictions to be spliced. |
+| `StartUtc` | string | optional | Start date of the spliced interval in UTC timezone in ISO 8601 format. The time must be the midnight of the day when converted to enterprise's local time. |
```

Comment:
I think it is better not to use the word "midnight" because it is ambiguous ("23:59:59" is midnight as much as "00:00:00"), better to specify time "00:00:00". And we can reference [Datetime](../guidelines/serialization.md#datetimes) for further guidance.